model/SUBDD.f

```diff
@@ -1773,6 +1773,13 @@ subroutine parse_subdd
       input_sizes3(k) = lm
       call tijph_defs(diaglists(1,k),nmax_possible,diaglens(k))
 #endif
+#ifdef TRACERS_GC
+      k = k + 1
+      catshapes(k) = 'aijlh'; categories(k) = 'taijlh'
+      input_sizes3(k) = lm
+      call tijlh_defs(diaglists(1,k),nmax_possible,diaglens(k))
+#endif
```

Comment:
Adds the ability to have GEOS-Chem tracers outputted at hourly timescales.

Backbone for how to translate #23.